src/openpersonen/api/tests/test_data/ingeschreven_persoon.py

```diff
@@ -121,7 +121,7 @@
                         "maand": 1,
                     },
                     "nationaliteit": {"code": None, "omschrijving": None},
-                    "redenOpname": {"code": None, "omschrijving": None},
+                    "redenOpname": {"code": None, "omschrijving": "Onbekend"},
```

Comment:
I'd expect `code` to be `"."` here, since that's the fallback value according to the model:
```suggestion
                    "redenOpname": {"code": ".", "omschrijving": "Onbekend"},
```